packages/EasyTest/src/Traits/MessengerAssertionsTrait.php

```diff
@@ -266,11 +266,20 @@ private static function runMessengerWorker(
         foreach ($transport->getRejected() as $envelope) {
             /** @var \Symfony\Component\Messenger\Stamp\ErrorDetailsStamp $errorDetailsStamp */
             foreach ($envelope->all(ErrorDetailsStamp::class) as $errorDetailsStamp) {
-                foreach ($expectedExceptions as $key => $expectedExceptionPair) {
+                foreach ($expectedExceptions as $key => $expectedExceptionPairOrClass) {
+                    // if $expectedExceptionPairOrClass is a string = exceptionClass
+                    if ($expectedExceptionPairOrClass === $errorDetailsStamp->getExceptionClass() &&
```

Comment:
Do we need to add `\is_string($expectedExceptionPairOrClass)`?

Reply:
No, strict comparison already do it

Reply:
I got it. Thanks.